CMake Install Error when not building Tests

When using `-DBUILD_TESTING=OFF`, `make install` fails with:

```
CMake Error at cmake_install.cmake:124 (file):
  file INSTALL cannot find "/home/vavolkl/repo/ced/build/bin/test_ced": No
  such file or directory.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -127,20 +127,18 @@ ENDIF( CED_SERVER )
 
 
 # ------------------ CED TESTS ------------------
+IF( BUILD_TESTING )
 #FOREACH( _testname test_ced test_ced_mhits test_ced_color )
-FOREACH( _testname test_ced test_ced_mhits )
+  FOREACH( _testname test_ced test_ced_mhits )
 
-    IF( BUILD_TESTING )
-        ADD_EXECUTABLE( ${_testname} ./src/tests/${_testname}.cc )
-    ELSE()
-        ADD_EXECUTABLE( ${_testname} EXCLUDE_FROM_ALL ./src/tests/${_testname}.cc )
-    ENDIF()
+    ADD_EXECUTABLE( ${_testname} ./src/tests/${_testname}.cc )
 
     TARGET_LINK_LIBRARIES( ${_testname} CED )
 
     INSTALL( TARGETS ${_testname} DESTINATION bin )
 
-ENDFOREACH()
+  ENDFOREACH()
+ENDIF()
 
 
 

